donate/templates/payment/card_master.html

```diff
@@ -74,8 +74,12 @@ <h1 class="heading heading--primary heading--bottom-margin">{% trans "Your Detai
                             {% render_form_field form.last_name %}
                             {% render_form_field form.address_line_1 %}
                             <div class="form__group form__group--city-post">
-                                {% render_form_field form.post_code %}
-                                {% render_form_field form.city %}
+                                <div class="post-code-input-container">
```

Comment:
Wrapped these fields in respective divs so I can add the "hidden"/"full-width" class when neccesary

Reply:
This should not be necessary: the `render_form_field` tag lets you specify additional custom classes, if needed (see https://github.com/Tivix/django-common/blob/407d208121011a8425139e541629554114d96c18/django_common/templatetags/custom_tags.py#L50-L73)

Reply:
My bad - this is our own tag in `donate/core/templatetags/form_tags.py`, so we can just update that so that it can do additional classes.